app/javascript/lib/validation/validate-file-content.js

```diff
@@ -22,7 +22,8 @@ import {
 
 import getSCPContext from 'providers/SCPContextProvider'
 
-// from lib/assets/metadata_schemas/alexandria_convention_schema.json (which in turn is from scp-ingest-pipeline/schemas)
+// from lib/assets/metadata_schemas/alexandria_convention_schema.json
+// (which in turn is from scp-ingest-pipeline/schemas)
 export const REQUIRED_CONVENTION_COLUMNS = [
```

Comment:
Just thinking out loud here - we have an existing API endpoint at https://singlecell.broadinstitute.org/single_cell/api/v1/metadata_schemas/alexandria_convention/latest/json that will serve up the current version of the Alexandria convention.  There's a property at the end of the JSON schema file called `required` that we could simply consume, rather than hard-coding.  Just a thought...

Reply:
Agreed, I think there's consensus it'd be cleaner.  Refactoring along those things seems most opportune once CSFV handles ontology validation (which I think is feasible to do quite performantly).